src/Symfony/Component/FeatureFlag/Debug/TraceableFeatureChecker.php

```diff
@@ -0,0 +1,43 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\FeatureFlag\Debug;
+
+use Symfony\Component\FeatureFlag\DataCollector\FeatureCheckerDataCollector;
+use Symfony\Component\FeatureFlag\FeatureCheckerInterface;
+
+final class TraceableFeatureChecker implements FeatureCheckerInterface
+{
+    public function __construct(
+        private readonly FeatureCheckerInterface $decorated,
+        private readonly FeatureCheckerDataCollector $dataCollector,
+    ) {
+    }
+
+    public function isEnabled(string $featureName, mixed $expectedValue = true): bool
+    {
+        $isEnabled = $this->decorated->isEnabled($featureName, $expectedValue);
```

Comment:
That would leverage the default value of the decorated object instead of forcing `$expectedValue` to `true`. If we change it, this needs a comment.

https://3v4l.org/RgFiN

Reply:
Traceable are expected to do their job seamlessly - `$expectedValue` must be the same with or without debug mode. So 👍 for what @94noni suggests